homeassistant/components/nissan_leaf/__init__.py

```diff
@@ -499,16 +462,16 @@ def device_state_attributes(self):
             'last_attempt': self.car.last_check,
             'updated_on': self.car.last_battery_response,
             'update_in_progress': self.car.request_in_progress,
-            'location_updated_on': self.car.last_location_response,
             'vin': self.car.leaf.vin,
         }
 
-    async def async_added_to_hass(self):
+    def added_to_hass(self):
```

Comment:
Revert this. This method is always a coroutine.

Reply:
Thanks.  Fixed in commit https://github.com/home-assistant/home-assistant/pull/21145/commits/a68e5602096d15159dbd8d79f0176e9c351d7bd1